### PR TITLE
custom target paths for storing artefacts

### PR DESCRIPTION
**Pull request by dejii**
Related issue: https://github.com/re-data/re-data/issues/236

### CODE REVIEW

File: re_data/command_line.py [hunk 1]
```diff
@@ -299,21 +340,38 @@ def serve(port):
...
+        Defaults to the 'target-path' used in dbt_project.yml
+    """
+)
 @add_options(dbt_flags)
-def slack(start_date, end_date, webhook_url, subtitle, **kwargs):
+def slack(start_date, end_date, webhook_url, subtitle, re_data_target_dir, **kwargs):
     start_date = str(start_date.date())
     end_date = str(end_date.date())
+
+    project_root = os.getcwd() if not kwargs.get('project_dir') else os.path.abspath(kwargs['project_dir'])
```
Comment (mateuszklimek) [inline re_data/command_line.py:356]: it does seem to be quite repeated? Refactoring that to a function?

File: getting_started/toy_shop/packages.yml [hunk 2]
```diff
@@ -2,5 +2,5 @@ packages:
     - package: dbt-labs/dbt_utils
       version: [">=0.7.0", "<0.9.0"]
     
-    - package: re-data/re_data
-      version: [">=0.6.0", "<0.7.0"]
\ No newline at end of file
+    - git: https://github.com/re-data/dbt-re-data.git
```
Comment (mateuszklimek) [inline getting_started/toy_shop/packages.yml:5]: before merging, let's change it back to package version

File: re_data/command_line.py (hunk 1 above)
Comment (dejii) [reply re_data/command_line.py]: this is done now.

File: getting_started/toy_shop/packages.yml (hunk 2 above)
Comment (mateuszklimek) [reply getting_started/toy_shop/packages.yml:5]: @dejii but it's not changed?

**Pull request merged**